pkg/auth/authproviders/backend_factory.go

```diff
@@ -8,7 +8,7 @@ import (
 // BackendFactory is responsible for creating Backends.
 type BackendFactory interface {
 	// CreateBackend creates a new backend instance for the given auth provider, using the specified configuration.
```

Comment:
Please explain what `mappings` mean here. The comment will likely duplicate some info from the comment you will write in .proto and that's great. Specifically, I suggest to mention the intent (forwarding specific claims from IdP to Rox token) and the format (key is path and value is attribute name).

Alternatively, you can create a type, e.g., `IdPToRoxClaimMapping`, where all the comments above will go. Up to you.